etl/steps/data/meadow/technology/2023-03-06/mobile_money.py

```diff
@@ -0,0 +1,46 @@
+"""Load a snapshot and create a meadow dataset."""
+
+import pandas as pd
+from owid.catalog import Table
+from structlog import get_logger
+
+from etl.helpers import PathFinder, create_dataset
+from etl.snapshot import Snapshot
+
+# Initialize logger.
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("mobile_money.start")
+
+    #
+    # Load inputs.
+    #
+    # Retrieve snapshot.
+    snap: Snapshot = paths.load_dependency("mobile_money.xlsx")
+
+    # Load data from snapshot.
+    df = pd.read_excel(snap.path, sheet_name="Accounts", usecols="B:BK", skiprows=25, nrows=16)
+
+    # Reshape to tidy format and rename cols
+    df = df.melt(id_vars="Regions", var_name="year", value_name="active_accounts_90d").rename(
+        columns={"Regions": "entity"}
```

Comment:
We usually use the name "country" (or in some cases "region") for columns of countries and regions. The word "entity" is often just used in the grapher step, for historical reasons. But if you prefer to call it "entity", that's fine.